coc.md

```diff
@@ -4,81 +4,72 @@ description: Guidelines on how we expect our members to behave.
+In this document, we describe our intentions for Fosstodon, and set our expectations for all users on Fosstodon.
 
-## Goals
+This is a living document, which may be amended at any time by the [Fosstodon team](https://hub.fosstodon.org/team) to better communicate or adjust these goals.
 
-* Provide a friendly, safe, and welcoming environment for everyone interested in Free/Libre/Open-Source Software, regardless of level of experience, gender identity or expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, religion, nationality, political affiliation, or other similar characteristic.
-* Create a supportive community, where people freely share their passions and are excited and positive about each other's successes.
-* Provide a venue for casual conversations — we may all be passionate about technology, but that doesn't mean that our conversations always centre on that shared passion.
+### What we are
 
-## Non-Goals
+Fosstodon is a Mastodon instance that federates with many other servers in the fediverse, also known as the open social web. The theme of this instance is Free/Libre/Open-Source software and adjacent communities. This is what the "FOSS" in our name represents.
 
-While it's important to know what Fosstodon is, it's just as important to understand what we're not. Fosstodon is **not**:
+### Intentions
 
-* A free speech zone. Fosstodon is a community with specific — and high — standards of behaviour. If you would like to exercise your full free speech rights, you can do so **somewhere else**; while on Fosstodon, you should agree to follow the community rules set out in this CoC.
-* An advocacy space. There are many important, controversial and political issues; and advocating for what you believe is important. However, Fosstodon is not the place for that advocacy. Even correct views can violate our CoC.
-* An experts-only space. We all share a love of FOSS, but no Fosstodon user should be shamed for not understanding a basic concept or told to read the manual before asking questions about a certain technology.
+Fosstodon intends to provide a friendly environment for those who want a social media experience that prioritizes FOSS, where you can share your interests, passions, and achievements. Nobody should feel unwelcome on the basis of their skills, experience, or ability to read a manual; Fosstodon is *not* an experts-only space.
 
-## Public Toots
+While using Fosstodon or engaging with Fosstodon members, everyone should be safe from harassment and similar antisocial behavior. As a community, we aim to curate a safe place for anyone, regardless of their characteristics including but not limited to: race, ethnicity, national identity, spoken language, sex, gender identity or expression, sexuality, sexual orientation, physical or mental health status, disability, other body characteristics, or age.
```

Comment:
This could be made less wordy and more maintainable:
```suggestion
While using Fosstodon or engaging with Fosstodon members, everyone should be safe from harassment and similar antisocial behavior. As a community, we aim to curate a safe place for everyone.
```
But I understand if you prefer to keep this incomplete/growing list of characteristics if it will make certain classes of users more comfortable.